service_info_cms/views.py

```diff
@@ -0,0 +1,31 @@
+from cms.models.pagemodel import Page
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from django.shortcuts import redirect, render
+from .models import PageRating
+
+
+def update_page_rating(request):
+    if request.method == 'POST':
+        rating = int(request.POST.get("rating", ""))
+        page_id = int(request.POST.get("page_id", ""))
+        return_url = request.POST.get("return_url", "")
+        page = Page.objects.get(pk=page_id)
+        page_rating, created = PageRating.objects.get_or_create(page_obj=page)
+        page_rating.update_rating_average(rating)
+        return redirect(return_url)
+
+
+def page_ratings(request):
+    """Listing of all page ratings."""
```

Comment:
If I were in Omar or Hanaa's place I'd want to use admin to see page ratings instead of having a new admin-like interface.  There it is trivial to enable sorting, paging is free, style expectations are nil, etc.  Can you see what the rest of "us" thinks?

If this is kept, it needs to be @login_required.  I don't think random users should be able to see this.